src/odb/include/odb/db.h

```diff
@@ -7520,31 +7594,105 @@ class dbModInst : public dbObject
   static dbSet<dbModInst>::iterator destroy(dbSet<dbModInst>::iterator& itr);
 
   static dbModInst* getModInst(dbBlock* block_, uint dbid_);
-
-  std::string getName() const;
+  dbModITerm* getdbModITerm(dbId<dbModITerm>);
 
   std::string getHierarchicalName() const;
+
+  bool getPinAtIx(unsigned ix, dbModITerm*& ret) const;
+  bool findModITerm(const char* name, dbModITerm*& ret);
+
+  dbSet<dbModITerm> getModITerms();
+
   // User Code End dbModInst
 };
 
+class dbModITerm : public dbObject
+{
+ public:
+  void setFlags(uint flags);
+
+  uint getFlags() const;
+
+  dbModInst* getParent() const;
+
+  void setNet(dbModNet* net);
+
+  dbModNet* getNet() const;
+
+  void setNextNetModiterm(dbModITerm* next_net_moditerm);
+
+  dbModITerm* getNextNetModiterm() const;
+
+  void setPrevNetModiterm(dbModITerm* prev_net_moditerm);
+
+  dbModITerm* getPrevNetModiterm() const;
+
+  void setNextEntry(dbModITerm* next_entry);
+
+  dbModITerm* getNextEntry() const;
```

Comment:
private

Reply:
automatically generated code. Best left as is from generator.